lib/matplotlib/tests/test_font_manager.py

```diff
@@ -317,7 +317,7 @@ def test_get_font_names():
             font = ft2font.FT2Font(path)
             prop = ttfFontProperty(font)
             ttf_fonts.append(prop.name)
-        except:
+        except Exception:
```

Comment:
```suggestion
        except BaseException:
```

although I wonder why this is forgiving?

Reply:
Do we wish to catch `KeyboardInterrupt` and `SystemExit` here?

I would not have thought so.

I originally did this as a `noqa` comment, but changed to `except Exception:` at @timhoffm's [suggestion](https://github.com/matplotlib/matplotlib/pull/25147#discussion_r1100722163).

There are other linter rules that we do not select which disallow `Exception/BaseException`, but the pycodestyle (derived) rule E722 only deals with bare except clauses.

Alternatively could go back to ignoring that check, though I think generally speaking, it is a good thing to avoid bare exceptions. This was just a situation where I was simplifying the configuration and found one with only a small number of examples, so thought fixing it may be the better call.

Reply:
I retract my comment, just being too biased to "match old behavior as much as possible".